packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -0,0 +1,753 @@
+    this.unset('logoutTimer');
+
+    // Validate if the device is currently in a meeting and is configured to
+    // required inactivity enforcement.
+    if (!this.isInMeeting && this.config.enableInactivityEnforcement &&
+      this.isReachabilityChecked) {
+      if (this.isInNetwork) {
+        this.setLogoutTimer(this.inNetworkInactivityDuration);
+      }
+      else {
+        this.setLogoutTimer(this.intranetInactivityDuration);
+      }
+    }
+  },
+
+  /**
+   * Set the value of the logout timer for the current registered device.
+   *
+   * @param {number} duration - Value in seconds of the new logout timer.
+   * @returns {void}
+   */
+  setLogoutTimer(duration) {
+    this.logger.info('devices: setting logout timer');
+
+    if (!duration || duration <= 0) {
+      return;
+    }
+
+    // Setup user activity date event listener.
+    this.on('change:lastUserActivityDate', () => { this.resetLogoutTimer(); });
+
+    // Initialize a new timer.
+    this.logoutTimer = safeSetTimeout(() => {
+      this.webex.logout();
+    }, duration * 1000);
+  },
+
+  // Resource method members.
+
+  /**
+   * Create a device. This is currently a template method to be implemented
+   * later and is subject to change. No parameters are present in the template
+   * method to prevent a linting error.
+   *
+   * @param {Object} deviceObj - Device configuration object.
+   * @returns {Promise<void, Error>}
+   */
+  create() {
+    this.logger.info('devices: creating device');
+    this.logger.warn('devices: creating devices is not currently implemented');
+
+    return Promise.resolve();
+  },
+
+  /**
+   * Get a device by deviceId. This is currently a template method to be
+   * implemented later and is subject to change.
+   *
+   * @param {string} deviceId - deviceId of the target device to get.
+   * @returns {Promise<void, Error>}
+   */
+  get(deviceId) {
+    this.logger.info(`devices: getting device \`${deviceId}\``);
+    this.logger.warn('devices: getting devices is not currently implemented');
+
+    return Promise.resolve();
+  },
+
+  /**
+   * List devices associated with the current authenticated instance. This is
+   * currently a template method to be implemented later and is subject to
+   * change.
+   *
+   * @returns {Promise<Array<Object>, Error>}
+   */
+  list() {
+    this.logger.info('devices: listing devices');
+    this.logger.warn('devices: listing devices is not currently implemented');
+
+    return Promise.resolve([]);
+  },
+
+  /**
+   * Remove a device by deviceId. This is currently a template method to be
+   * implemented later and is subject to change.
+   *
+   * @param {string} deviceId - deviceId of the target device to remove.
+   * @returns {Promise<void, Error>}
+   */
+  remove(deviceId) {
+    this.logger.info(`devices: removing device \`${deviceId}\``);
+    this.logger.warn('devices: removing devices is not currently implemented');
+
+    return Promise.resolve();
+  },
```

Comment:
These methods should be somewhere else. This file deals with the current device. These methods are more towards managing a user's registered devices.

Reply:
I will remove these methods to help meet the scope of the original plugin.